index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Collision Summit 2023</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <header id="social-header">
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href=""><img src="images/social/twitter-16.png" alt="twitter"></a></li>
+          <li><a href=""><img src="images/social/facebook-16.png" alt="facebook"></a></li>
+          <li><a href="#logout">Logout</a></li>
+          <li><a href="#mypage"> My Page</a></li>
+          <li><a href="#Portuguese"> Portuguese</a></li> 
+        </ul>
+      </nav>
+    </header>
+    <header>
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"   
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact">News</a></li>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164409139-a6cc0770-c867-4849-a7d2-0f0ff57d3ce1.png) | ![image](https://user-images.githubusercontent.com/10332499/164409054-201844d4-4259-4bd6-ace8-185efef2187e.png)

Reply:
resolved